pyproject.toml

```diff
@@ -164,6 +164,8 @@ percy = "^2.0.2"
 h5py = "^3.1.0"
 h5netcdf = "^0.11.0"
 pytest-xdist = "^2.2.1"
+GDAL = { version = "^3.3.2", optional = false, markers = "platform_system != 'Windows'"  }
+wradlib = { version = "^1.11.0", optional = false, markers = "platform_system != 'Windows'"  }
```

Comment:
I removed those entries. I added them after I saw that the pipeline failed because wradlib was not installed. But it still fails and I am not sure how to fix it.